website/src/logged_in/prediction_tool/CreatePrediction.js

```diff
@@ -178,7 +161,21 @@ function CreatePrediction(props) {
                                                 <MenuItem value={0}>No</MenuItem>
                                             </Select>
                                         </FormControl>
+                                    </Grid>
+
+                                    <Grid item xs={1} />
+
+                                    <Grid item xs={6}>
 
+                                    <Grid item xs={12} className={props.classes.inputPadTop}>
```

Comment:
Fix indentation here. Please ensure your code style is consistent with the existing codebase's style.

Reply:
This comment is widely applicable through the PR. Fix all style violations.